.drone/drone.yml

```diff
@@ -120,6 +120,51 @@ volumes:
 depends_on:
 - check
 kind: pipeline
+name: docker-main-commit
+platform:
+  arch: amd64
+  os: linux
+steps:
+- commands:
+  - git fetch origin --tags
+  image: golang:1.19
+  name: fetch-tags
+  volumes:
+  - name: gopath
+    path: /go
+- commands:
+  - if [ ! -f linux-amd64/helm ]; then
+  - '  wget -q https://get.helm.sh/helm-v3.9.0-linux-amd64.tar.gz'
+  - '  tar -zxvf helm-v3.9.0-linux-amd64.tar.gz'
+  - '  rm -f helm-v3.9.0-linux-amd64.tar.gz'
+  - fi
+  - cp linux-amd64/helm /usr/local/bin/helm
+  - make static
+  image: golang:1.19
+  name: static
+  volumes:
+  - name: gopath
+    path: /go
+- image: plugins/docker
+  name: container
+  settings:
+    password:
+      from_secret: dockerhub_password
+    repo: grafana/tanka
+    tags:
+    - ${DRONE_COMMIT}
```

Comment:
if possible it would be better to use ``git describe --tags``, to give the version numbers a bit more readability

```
laney@sherwood (main|✔)> git describe --tags
v0.23.0-1-g904a7c5
```

Reply:
Yeah, that's better indeed

Reply:
Maybe prefix with "main" or something to make it clear what they are